kubernetes/schema_volume_source.go

```diff
@@ -218,6 +218,41 @@ func commonVolumeSources() map[string]*schema.Schema {
 				},
 			},
 		},
+		"csi": {
+			Type:        schema.TypeList,
+			Description: "Represents a CSI Volume. More info: http://kubernetes.io/docs/user-guide/volumes#csi",
+			Optional:    true,
+			MaxItems:    1,
+			Elem: &schema.Resource{
+				Schema: map[string]*schema.Schema{
+					"driver": {
+						Type:        schema.TypeString,
+						Description: "the name of the volume driver to use. More info: https://kubernetes.io/docs/concepts/storage/volumes/#csi",
+						Required:    true,
+					},
+					"volume_handle": {
+						Type:        schema.TypeString,
+						Description: "A map that specifies static properties of a volume. More info: https://kubernetes.io/docs/concepts/storage/volumes/#csi",
```

Comment:
This description is incorrect - should refer to a string like:
`A string value that uniquely identifies the volume. `

Reply:
Fixed